e2e/suites/actions/delete.test.ts

```diff
@@ -50,8 +50,6 @@ describe('Delete content', () => {
         apis.admin.trashcan.emptyTrash().then(done);
     });
 
-    xit('');
```

Comment:
there is a known  bug with the reporter and nested describe without ```it``` and this is a workaround.